utility/staticVariables.py

```diff
@@ -9,3 +9,5 @@
 json_content_type = {'Content-type': json_header_str}
 json_accept = {'Accept': json_header_str}
 json_headers = {**json_content_type, **json_accept}
+
+date_format = '%Y-%m-%d_%H:%M:%S-UTC'
```

Comment:
We have several date formats through project, I'd rather name it _backup_date_format_ or something like that.